### PR TITLE
Improve example context

### PR DESCRIPTION
**Pull request by matthieu88160**
This commit add the component standalone examples and place the existing ones into a controller context. (Framework example first)

### CODE REVIEW

**Review by javiereguiluz — APPROVED** [review]
:+1:

File: form/action_method.rst [hunk 1]
```diff
@@ -9,31 +9,116 @@ URL under which the form was rendered. Sometimes you want to change these
...
+        use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+        use Symfony\Component\Form\Extension\Core\Type\TextType;
+        use Symfony\Component\Form\Extension\Core\Type\DateType;
+        use Symfony\Component\Form\Extension\Core\Type\SubmitType;
+
+        class DefaultController extends Controller
+        {
+            public function newAction()
+            {
+
```
Comment (HeahDude) [inline form/action_method.rst]: This blank line is useless.

File: form/action_method.rst [hunk 2]
```diff
@@ -9,31 +9,116 @@ URL under which the form was rendered. Sometimes you want to change these
...
+                // ...
+
+                $form = $this->createFormBuilder($task)
+                    ->setAction($this->generateUrl('target_route'))
+                    ->setMethod('GET')
+                    ->add('task', TextType::class)
```
Comment (HeahDude) [inline form/action_method.rst]: This PR should target 2.7 and use type names (i.e `'text'`).

**Comment by xabbuh** [comment]
Thank you @matthieu88160.

**Pull request closed without merging**